contracts/verifiers/ZKPVerifier.sol

```diff
@@ -14,28 +14,24 @@ contract ZKPVerifier is IZKPVerifier, Ownable {
      */
     uint256 public constant REQUESTS_RETURN_LIMIT = 1000;
 
-    // This empty reserved space is put in place to allow future versions
-    // of the ZKPVerifier contract to inherit from other contracts without a risk of
-    // breaking the storage layout. This is necessary because the parent contracts in the
-    // future may introduce some storage variables, which are placed before the ZKPVerifier
-    // contract's storage variables.
-    // (see https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable#storage-gaps)
-    // slither-disable-next-line shadowing-state
-    // slither-disable-next-line unused-state
-    uint256[500] private __gap_before;
-
-    // msg.sender-> ( requestID -> is proof given )
-    mapping(address => mapping(uint64 => bool)) public proofs;
-
-    mapping(uint64 => IZKPVerifier.ZKPRequest) internal _requests;
+    /// @dev Main storage structure for the contract
+    struct MainStorage {
+        // msg.sender-> ( requestID -> is proof given )
+        mapping(address => mapping(uint64 => bool)) proofs;
```

Comment:
how to get an access to this map from parent contract? Maybe it makes sence to add getters